### PR TITLE
1509 task6 getting started with aws

### PR DESCRIPTION
**Pull request by mjunaid-cnc**

### CODE REVIEW

File: Task6-AWS/Function.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    /// <summary>
+    /// This method is called for every Lambda invocation. This method takes in an S3 event object and can be used 
+    /// to respond to S3 notifications.
+    /// </summary>
+    /// <param name="evnt"></param>
+    /// <param name="context"></param>
+    /// <returns></returns>
+    public async Task FunctionHandler(S3Event evnt, ILambdaContext context)
+    {
+        UserRepository userRepository = new UserRepository();
```
Comment (sikandar-codeninja) [inline Task6-AWS/Function.cs]: what is this object for?

Comment (mjunaid-cnc) [reply Task6-AWS/Function.cs]: This line of code is redundant. I'm not using it.

Comment (sikandar-codeninja) [reply Task6-AWS/Function.cs]: then u can remove the un-used code

Comment (mjunaid-cnc) [reply Task6-AWS/Function.cs]: Done

File: Task6-AWS/Function.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                var file = await this.S3Client.GetObjectAsync(s3Event.Bucket.Name, s3Event.Object.Key);
+                if (!file.Key.StartsWith("processed_"))
+                {
+                    using var reader = new StreamReader(file.ResponseStream);
+                    var fileContents = await reader.ReadToEndAsync();
+                    LambdaLogger.Log(fileContents);
+                    var user = JsonConvert.DeserializeObject<User>(fileContents);
+                    if (user != null)
+                    {
+                        var connectionString = $"Server=database-1.c1jusiebdoer.eu-north-1.rds.amazonaws.com;Database=Task6;User ID=admin;Password=everton12;";
```
Comment (sikandar-codeninja) [inline Task6-AWS/Function.cs]: this should be fetched from AWS Secret Manager by using your DataHelper.GetConnectionString()

Comment (mjunaid-cnc) [reply Task6-AWS/Function.cs]: Done

**Review by sikandar-codeninja — APPROVED** [review]

**Pull request merged**